x-pack/plugin/searchable-snapshots/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/indices_stats.yml

```diff
@@ -154,3 +154,12 @@ teardown:
   - gt: { indices.frozen-docs.primaries.segments.file_sizes.si.max_size_in_bytes: 0 }
   - gt: { indices.frozen-docs.primaries.segments.file_sizes.si.average_size_in_bytes: 0 }
   - match: { indices.frozen-docs.primaries.segments.file_sizes.si.description: "Segment Info" }
+
+  - do:
+      indices.delete:
+        index: cold-docs
```

Comment:
NIT in case my comment below does not make sense: I think `index: [cold-docs, frozen-docs]` should work here as well, no need for separate delete calls?